src/resources/postcss/views/_loader.pcss

```diff
@@ -0,0 +1,24 @@
+.tribe-events-container {
+	position: relative;
+
+	.tribe-events-view-loader {
+		align-content: center;
+		justify-content: center;
+		display: flex;
+		position: absolute;
+		top: 0;
+		right: 0;
+		bottom: 0;
+		left: 0;
+		z-index: 25000;
+		background: rgba( 255, 255, 255, 0.6 );
+
+		&.tribe-hidden {
+			@mixin hidden;
+		}
+
+		.tribe-events-view-loader-spinner {
```

Comment:
this should be `.tribe-events-view-loader__spinner` and not wrapped in `.tribe-events-view-loader`.